dnf5daemon-client/commands/repoquery/repoquery.cpp

```diff
@@ -135,29 +202,86 @@ void RepoqueryCommand::run() {
         options.insert(std::pair<std::string, std::vector<std::string>>("package_attrs", {"full_nevra"}));
     }
 
-    dnfdaemon::KeyValueMapList packages;
```

Comment:
Note to `package_attrs`  that are defined above - The output is used for  package wrapper that has set of supported attributes. When data are not available then it raises Out of range error. What about to store required attribute directly in the class?

Reply:
I know about this issue. But dnf5daemon-client is kind of an example how to work with D-Bus API and never got priority high enough to work on fixing these edgecases. I can file a low priority issue on it.